TFT/src/User/Configuration.h

```diff
@@ -56,20 +56,20 @@
 #define SPEED_ID         {"Sp.", "Fr."}
 
 // Default move speed mm/min
-#define DEFAULT_SPEED_MOVE      3000
+#define DEFAULT_SPEED_MOVE      3600
 
 // Extrude speed mm/min
 #define EXTRUDE_SLOW_SPEED      60
 #define EXTRUDE_NORMAL_SPEED    600
 #define EXTRUDE_FAST_SPEED      1200
 
 // Size of machine
-#define X_MIN_POS 0
+#define X_MIN_POS -10

```

Comment:
may cause problem to other type of printer
```suggestion
#define X_MIN_POS 0
```